paddle/fluid/inference/analysis/ir_pass_manager.cc

```diff
@@ -99,9 +99,15 @@ void IRPassManager::CreatePasses(Argument *argument,
                 new std::unordered_set<std::string>());
 #ifdef PADDLE_WITH_MKLDNN
     } else if (pass_name == "cpu_quantize_placement_pass") {
-      pass->Set("quantize_enabled_op_types",
-                new std::unordered_set<std::string>(
-                    argument->quantize_enabled_op_types()));
+      if (!argument->int8_enabled_op_types().empty()) {
```

Comment:
Hi @wozna, can you help to see if this is the correct way to handle it?

Reply:
> Hi @wozna, Do you have any suggestions for PR-CI-Windows failures? Thanks~

We should set default int8_enabled_op_types as not empty list like before or if we want by default to not quantize anything then we need to change unit tests

Reply:
> > Hi @wozna, Do you have any suggestions for PR-CI-Windows failures? Thanks~
> 
> We should set default int8_enabled_op_types as not empty list like before or if we want by default to not quantize anything then we need to change unit tests

@baoachun  @sfraczek Yes the problem is that this `cpu_quantize_placement_pass` is also used in PTQ , quant2_int8_mkldnn_pass and unittests. So when you moved this `op_types` checking from pass to `AnalysisConfig::EnableMkldnnInt8` , mechanism when op_types is empty or is not set works only for this EnableMkldnnInt8. I'm thinking if maybe better solution will be to move `quantize_enabled_op_types_`  back to cpu_quantize_placement_pass